smoke-tests/src/test/java/com/splunk/opentelemetry/ProfilerSmokeTest.java

```diff
@@ -156,26 +164,24 @@ void verifyIngestedLogContent() throws Exception {
         .pollInterval(1, TimeUnit.SECONDS)
         .untilAsserted(() -> assertThat(countThreadDumpsInFakeBackend()).isGreaterThan(0));
 
-    List<ExportLogsServiceRequest> resourceLogs = fetchResourceLogs();
-    List<LogRecord> logs = flattenToLogRecords(resourceLogs);
-    assertAllThreadDumps(logs, log -> 1001 == getLongAttr(log, "source.event.period"));
-    assertAllThreadDumps(logs, log -> "otel.profiling".equals(getStringAttr(log, "sourcetype")));
-
-    Optional<LogRecord> jfrThread =
-        logs.stream()
-            .filter(log -> log.getBody().getStringValue().startsWith("\"Catalina-utility-1\""))
-            .findFirst();
-    assertThat(jfrThread).isNotEmpty();
-    Optional<LogRecord> mainThread =
-        logs.stream()
-            .filter(log -> log.getBody().getStringValue().startsWith("\"main\""))
-            .findFirst();
-    assertThat(mainThread).isNotEmpty();
-    assertThat(countTLABs(logs)).isGreaterThan(0);
-    assertAllThreadDumps(justTLABs(logs), log -> getLongAttr(log, "memory.allocated") > 0);
-    assertAllThreadDumps(justTLABs(logs), log -> log.getBody().getStringValue().startsWith("\""));
-    assertAllThreadDumps(
-        justTLABs(logs), log -> log.getBody().getStringValue().split("\n").length > 2);
+    LogsInspector logs = telemetryRetriever.waitForLogs();
+
+    assertThat(logs.getThreadDumpEvents())
+        .isNotEmpty()
+        .allMatch(log -> 1001 == getLongAttr(log, "source.event.period"))
+        .allMatch(log -> "otel.profiling".equals(getStringAttr(log, "com.splunk.sourcetype")));
```

Comment:
Turns out some of these assertions were broken (attribute names were changed), so I fixed them too.